### ISSUE TITLE
Word cloud enhancement

### ISSUE THREAD

**Issue by trevore** [oculusinfo/TellFinder#22]
IJM analyst would like to select word in word cloud and see which ad the words show up in.  Better yet, would like to full text search within tagged ads or ad cluster.

PILOT-42

**Comment by eric-oculus** [comment]
Agreed this would be useful. It will be tricky since we calculate the word cloud on the server. We would either need to:
  1) Switch to a client-side word cloud rendering (funnily enough, I saw just such a demo 10min ago)
  2) Send a complicated server call so that it can determine the word selected (less good)
